src/main/resources/application-local.yml

```diff
@@ -14,8 +11,34 @@ spring:
         show-sql: true
         format_sql: true
 
+  flyway:
+    enabled: false
+    placeholders:
+      email: ${FLYWAY_EMAIL}
+      password: ${FLYWAY_PASSWAORD}
+      role: ${FLYWAY_ROLE}
+      enabled: ${FLYWAY_ENABLED}
+
 logging:
   level:
     root: INFO
     com:
-      khutircraftubackend: DEBUG
\ No newline at end of file
+      khutircraftubackend: DEBUG
+  file:
+    name: logs/latest.log
+
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "info, loggers, health, auditevents, logfile, metrics, sessions"
+      base-path: /adminac
```

Comment:
better name it `/actuator` 

here is the example: 
https://www.digitalocean.com/community/tutorials/spring-boot-actuator-endpoints